### PR TITLE
prow/pr_status: get user login after exchange the token

### PR DESCRIPTION
**Pull request by qhuynh96**
* This fixes pr status does not has the user's login to request after being redirected back from authentication

### CODE REVIEW

File: prow/githuboauth/githuboauth.go [hunk 1]
```diff
@@ -45,6 +56,12 @@ type OAuthClient interface {
 	AuthCodeURL(state string, opts ...oauth2.AuthCodeOption) string
 }
 
+type GithubClientGetterImpl struct{}
```
Comment (stevekuznetsov) [inline prow/githuboauth/githuboauth.go]: Usually these are not exported and instead there is:

```go
func NewGitHubClientGetter() GitHubClientGetter {
    return &githubClientGetter{}
}
```

**Comment by BenTheElder** [comment]
@stevekuznetsov @cjwagner any last thoughts?

**Comment by stevekuznetsov** [comment]
No, 

/lgtm

**Pull request merged**